public/app/features/panel/panel_menu.js

```diff
@@ -14,6 +14,7 @@ function (angular, $, _, Tether) {
           '<span class="panel-title drag-handle pointer">' +
             '<span class="icon-gf panel-alert-icon"></span>' +
             '<span class="panel-title-text drag-handle">{{ctrl.panel.title | interpolateTemplateVars:this}}</span>' +
+            '<span class="panel-help-text"><info-popover mode="bold">{{ctrl.panel.helpText}}</info-popover></span>' +
```

Comment:
Other option could be to use:
```<i class="fa fa-info-circle" bs-tooltip="ctrl.panel.helpText">```
and to use a couple of ng-if statements or something I think!